index.js

```diff
@@ -53,13 +63,17 @@ NYC.prototype._createInstrumenter = function () {
 NYC.prototype.addFile = function (filename, returnImmediately) {
   var instrument = true
   var relFile = path.relative(this.cwd, filename)
+  var include
 
-  // only instrument a file if it's not on the exclude list.
   for (var i = 0, exclude; (exclude = this.exclude[i]) !== undefined; i++) {
+    include = this.include[i]
```

Comment:
@Lalem001 good find, perhaps you could add a couple tests @davidchase -- was going to bug you about this already.

I haven't forgotten about this pull request, I've been doing a big overhaul of nyc that makes it work properly with ES6 :) I hope to have a few releases of the project out this weekend.